elliott/elliottlib/bzutil.py

```diff
@@ -259,6 +261,8 @@ def security_level(self):
             return None
 
     def is_tracker_bug(self):
+        if self.bug.fields.issuetype.name == "Vulnerability":
+            return True
```

Comment:
Let's not do this.. we have learned from experience that bugs can be created with missing metadata, so let's verify all metadata that we need in this new type and return True only when all of them are present

Reply:
the new "Vulnerability" is designed to can't be created or modified except ProdSec, so when we see such type we can trust it.